### ISSUE TITLE
Media: detail screen navigation buttons are not accessible and image and editable fields need hints

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12878]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > Media.
1. Select an image and double tap to open media details.
1. Swipe right to navigate through all the elements on screen.

**Result:** [32s](https://cloudup.com/cbXD05KPrh2)

* Navigation buttons are not accessible.
* Image should be marked as a button with a proper hint.
* It would be good to add hints to editable fields.

<img src="https://user-images.githubusercontent.com/1119271/68257817-cf33f200-fff1-11e9-9cc3-cc435fd11a28.png" width="270" alt="media-detail-screen">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>
